docs/annotations.md

```diff
@@ -183,3 +184,37 @@ spec:
           serviceName: go-server-service
           servicePort: 80
 ```
+
+## Use Private IP
+
+This annotation allows to specify whether to expose this endpoint on Private IP of Application Gateway.
```

Comment:
This sentence is missing a pronoun it seems. Should this be `This annotation allows us to specify`. Aadding `us`, or `you` will help.